src/Acclaim/Provider.php

```diff
@@ -51,11 +51,11 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'       => $user['data']['id'],
+            'id' => $user['data']['id'],
             'nickname' => $user['data']['first_name'],
-            'name'     => $user['data']['first_name'],
```

Comment:
Could we leave the align arrays rule on? changes such as these just pollute the commit history imo.

Reply:
Sure 👍